etc/blocks/CollectRecent.cfg

```diff
@@ -4,7 +4,7 @@
         "hours": 0,
         "microseconds": 0,
         "minutes": 0,
-        "seconds": 0
+        "seconds": 1
```

Comment:
This should be `-1`, meaning the window will never expire. I've tested that this works on my end, and is consistent with the block readme: `A zero value means that all signals expire immediately.`

This is working  by coincidence because you're getting signals more frequently than every second from the mobile UI.